.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,28 @@
+## Description
+
+Provide a summary of the changes introduced by this pull request.
+
+## Related Issues
+
+Mention any related issues that are being addressed by this pull request.
+
+Fixes # (issue)
+
+## Screenshots (if applicable)
+
+Include screenshots or gifs to visually demonstrate the changes made, if applicable.
+
+## Checklist
+
+Make sure all items are checked before submitting the pull request. Remove any items that are not applicable.
+
+- [ ] I have used agoric's linter on my code (https://github.com/Agoric/agoric-sdk/discussions/8274)
+- [ ] I have updated the documentation to reflect the changes (if applicable).
+- [ ] I have added/updated unit tests to cover the changes.
+- [ ] All existing tests pass.
+- [ ] The PR title is clear and concise.
+
```

Comment:
maybe add "(changes in /frontend) `cd frontend && yarn build` runs successfully

Reply:
done!